toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -536,7 +536,8 @@ func convertWriteableFormatToOutputImage(ic *ImageCustomizerParameters, inputIso
 		}
 
 	case imagecustomizerapi.ImageFormatTypeCosi:
-		err := convertToCosi(ic)
+		err := convertToCosi(ic.buildDirAbs, ic.rawImageFile, ic.outputImageFile, ic.partUuidToFstabEntry,
```

Comment:
`ImageCustomizerParameters.outputImageBase` is no longer being used. So, it can be removed.